### DISCUSSION TITLE
Is it possible to use only OpenVAS Docker image?

### DISCUSSION THREAD

**Discussion by romelBen** [Secure-Compliance-Solutions-LLC/GVM-Docker#150] [General]
Hello everyone. I truly appreciate the work that is being done on this repo and thank Secure Compliance Solutions for supporting this. It means a lot to me and my team. I have a general question on the use of GVM 20.08 and would like to ask: Is it possible to **only** implement OpenVAS-Docker image rather than GVM-Docker image. I would to only use the scanning process and not the hub for GVM. Is this possible or would I need to use GVM? I would appreciate an explanation since I am new to GVM.

**Comment by pixelsquared** [comment]
Currently the OpenVAS-Docker image is only designed to work with the GVM-Docker image. Could you explain what your use case is for it?
